packages/callback-example/package.json

```diff
@@ -10,6 +10,7 @@
     "build:clean": "tsc --build --clean && tsc --build"
   },
   "dependencies": {
+    "@sphereon/common": "workspace:*",
```

Comment:
Sorry I probably should have been more clear. I meant `@sphereon/oid4vc-common`.

Reply:
done

Reply:
Okay let me go over the rest